app/views/components/_time-select.html.erb

```diff
@@ -12,19 +13,20 @@
     selectable[:checked] = true
   end
 %>
-<% if base_path && dates.length > 1 && current_selection %>
+<% if dates.length > 1 && current_selection %>
   <div class="app-c-time-select">
     <h2 class="govuk-heading-l"><%= t '.title', time_period: dates.find {|d| d[:value] == current_selection }[:text] %></h2>
     <%= render "govuk_publishing_components/components/details", {
       title: t('.change_dropdown')
     } do %>
-      <%= form_for('date', :url => "#{base_path}", :method => :get) do %>
+
         <%= render "govuk_publishing_components/components/radio", {
           name: "date_range",
           items: dates
         } %>
-        <%= render "govuk_publishing_components/components/button", {text: submit_button_text} %>
-      <% end %>
+        <% if render_button != false %>
+          <%= render "govuk_publishing_components/components/button", {text: submit_button_text} %>
+        <% end %>
```

Comment:
Should we just add the button to the form of the in `show.html.erb` rather than adding `render_button` - as the button only makes sense in when embedded in the form.

Reply:
We can't because it has to appear in the reveal and not outside of it - else you get the header "Last 30 days" and a submit button without seeing the rest of the form.

Reply:
Ah sure!